internal/utils/main.go

```diff
@@ -757,28 +795,42 @@ func CreateLedgerBackend(ctx context.Context, useCaptiveCore bool, env Environme
 	}
 
 	// Create ledger backend from datastore
-	fileConfig := ledgerbackend.LCMFileConfig{
-		StorageURL:        env.StorageURL,
-		FileSuffix:        ".xdr.gz",
-		LedgersPerFile:    1,
-		FilesPerPartition: 64000,
+	params := make(map[string]string)
+	params["destination_bucket_path"] = env.DatastorePath
+	dataStoreConfig := datastore.DataStoreConfig{
+		Type:   "GCS",
+		Params: params,
```

Comment:
Are there any other optional params allowed in `DataStoreConfig`? I'm just curious

Reply:
Not currently